packages/components/src/components/loaders/Spinner/Spinner.stories.tsx

```
@@ -9,45 +9,50 @@ const meta: Meta = {
-        ...getFramePropsStory(allowedSpinnerFrameProps).args,
-    },
-    argTypes: {
-        className: {
-            control: false,
-        },
-        ...getFramePropsStory(allowedSpinnerFrameProps).argTypes,
     },
+    argTypes,
 };
 
 export const Error: StoryObj<SpinnerProps> = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)